systems/juwels-mc.sh

```diff
@@ -0,0 +1,38 @@
+### environment ###
+
+# set up environment for building on the multicore part of juwels
+
+module load CMake/3.13.0
+
+module load Python/3.6.6
+ns_python=$(which python3)
+
+module load GCC/8.2.0 ParaStationMPI/5.2.1-1
+
+# for (core)neuron
+module load mpi4py/3.0.0-Python-3.6.6
+module load flex/2.6.4
+module load Bison/.3.1
+
+### compilation options ###
+
+ns_cc=$(which mpicc)
+ns_cxx=$(which mpicxx)
+ns_with_mpi=ON
+
+ns_arb_arch=skylake
```

Comment:
Use either `broadwell` or `skylake-avx512`.
`skylake` is for the consumer CPUs, not the Xeon.

Reply:
Done. Used `skylake-avx512`

Reply:
Which implications does that have on the benchmarking we did on JUWELS? We used the script to build Arbor for all the scaling runs.